boards/arm/nrf5340dk_nrf5340/nrf5340dk_nrf5340_cpuapp_ns.yaml

```diff
@@ -14,3 +14,4 @@ supported:
   - watchdog
   - usb_cdc
   - usb_device
+  - netif:openthread
```

Comment:
Why the same is not added in `nrf5340dk_nrf5340_cpuapp.yaml` (similarly for thingy53)?

Reply:
It was metioned here: https://github.com/zephyrproject-rtos/zephyr/pull/50986#discussion_r987791105

Reply:
For nRF9160 - agreed. But for nRF5340, I believe it is a valid use case to use openthread also without secure/non-secure split on the application core. And to be sure that both `nrf5340_cpuapp` targets are properly supported, both should be tested. I discussed it offline with @rlubos and we agreed on that. Thus please, add that capability also for secure targets in `nrf5340dk` and `thingy53`.